doc/src/fix_ave_moments.rst

```diff
@@ -0,0 +1,286 @@
+       f_ID = global scalar calculated by a fix with ID
+       f_ID[I] = Ith component of global vector calculated by a fix with ID, I can include wildcard (see below)
+       v_name = value calculated by an equal-style variable with name
+       v_name[I] = value calculated by a vector-style variable with name, I can include wildcard (see below)
+
+* one or more moments to compute can be listed
+* moment = *mean* or *stddev* or *variance* or *skew* or *kurtosis*, see exact definitions below.
+* zero or more keyword/arg pairs may be appended
+* keyword = *start* or *delay*
+
+  .. parsed-literal::
+
+       *start* args = Nstart
+         Nstart = invoke first after this time step
+       *history* args = Ndelay
+         Ndelay = keep a history of up to Ndelay invocations
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   fix 1 all ave/moments 1 1000 100 v_volume mean sttdev
+   fix 1 all ave/moments 1 200 1000 v_volume variance kurtosis history 10
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+Using one or more variables as input every few time steps, calculate the
+moments of the underlying distribution based on the samples collected
+over a time step window. The definitions of the moments calculated are
+given below.
+
+The group specified with this command is ignored.  However, note that
+specified values may represent calculations performed by computes and
+fixes which store their own "group" definitions.
+
+Each listed value can be the result of a :doc:`compute <compute>` or
+:doc:`fix <fix>` or the evaluation of an equal-style or vector-style
+:doc:`variable <variable>`.  In each case, the compute, fix, or variable
+must produce a global quantity, quantity, not a per-atom or local
+quantity.  If you wish to spatial- or time-average or histogram per-atom
+quantities from a compute, fix, or variable, then see the :doc:`fix
+ave/chunk <fix_ave_chunk>`, :doc:`fix ave/atom <fix_ave_atom>`, or
+:doc:`fix ave/histo <fix_ave_histo>` commands.  If you wish to sum a
+per-atom quantity into a single global quantity, see the :doc:`compute
+reduce <compute_reduce>` command.
+
+:doc:`Computes <compute>` that produce global quantities are those which
+do not have the word *atom* in their style name.  Only a few :doc:`fixes
+<fix>` produce global quantities.  See the doc pages for individual
+fixes for info on which ones produce such values.  :doc:`Variables
+<variable>` of style *equal* and *vector* are the only ones that can be
+used with this fix.  Variables of style *atom* cannot be used, since
+they produce per-atom values.
+
+The input values must all be scalars or vectors with a bracketed term
+appended, indicating the :math:`I^\text{th}` value of the vector is
+used.
+
+The result of this fix can be accessed as a vector, containing the
+interleaved moments of each input in order.  The first requested moment
+of input 1 has index 1, the second index 2, the first of input 2 has
+index 3 and so on.
```

Comment:
This is too simplistic I think.  I would say something like: If M moments are requested, then the moments of input 1 will be the first M values in the vector output by this fix.  The moments of input 2 will the next M values, etc.  If there are N values, the vector length will be N*M.